### ISSUE TITLE
Support for relative values?

### ISSUE THREAD

**Issue by nikkop** [metafizzy/zdog#52]
Hi 👋 
First of all thanks for an awesome library. Really like it so far!

I wonder if there are any plans on implementing functionality to work with relative values as well as absolute? Support both uppercase and lowercase of the canvas path commands, that is. 

Thanks
BR

**Comment by desandro** [comment]
**Add a 👍  reaction** to this issue if you would like to see this feature added. 

Yeah, this could be a nice feature. Currently path commands all use absolute values. And if you want to adjust a value it means re-calculating all the others.

**Comment by nikkop** [comment]
Yes, I noticed 😁. It can be done by each developer but it will be extremely cumbersome to calculate it.

How would we like the approach? Have a `relative` boolean for each shape? With `false` as default. 

I'd happily look into it an make a PR if the feature is wanted.
